src/main/daml/Daml/Finance/Util/Disclosure.daml

```diff
@@ -15,44 +14,29 @@ import Daml.Finance.Interface.Types.Common.Types (PartiesMap)
 import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (AddObservers(..), I, RemoveObservers(..), SetObservers(..))
 
 -- | Default implementation for `setObservers`.
--- The `refUpdate` argument is used to update the corresponding contract `Reference` and
--- can be set to `None` if your template does not have an accompanying `Reference` contract.
 setObserversImpl :
-  forall t i.
+  forall t.
```

Comment:
no need for `forall` here. It's only needed when using the type annotation operators (@t)